<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
  <title>
Deep Ecology - Do bacterial species exist?
  </title>
<!-- #feeds -->
<link href="//blog.byronjsmith.com/feeds/all.atom.xml" type="application/atom+xml" rel="alternate" title="Deep Ecology Full Atom Feed" />
<link href="//blog.byronjsmith.com/feeds/category/{slug}.atom.xml" type="application/atom+xml" rel="alternate" title="Deep Ecology Categories Atom Feed" />
<!-- /#feeds -->
  <link rel="stylesheet" type="text/css" href="//blog.byronjsmith.com/theme/css/main.css" />
  <link rel="stylesheet" type="text/css" href="//blog.byronjsmith.com/theme/css/pygment.css" />
<!-- #google-analytics-script -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-40659359-2', 'auto');
  ga('require', 'linkid', 'linkid.js');
  ga('send', 'pageview');

</script>
<!-- /#google-analytics-script -->


</head>

<body id="main">
<!-- #github-ribbon -->
<a href="https://github.com/bsmith89">
  <img style="position: absolute; top: 0; right: 0; border: 0;"
      src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67"
      alt="Fork me on GitHub"
      id="gh-ribbon" />
</a>
<!-- /#github-ribbon -->


  <div id="all-text">
    <header id="main-banner" class="banner">
<!-- page-chrome -->
<h1 id="blog-title">
  <a href="//blog.byronjsmith.com/">
    Deep Ecology
    <span id="blog-subtitle">
      A blog of the new microbiology.
    </span>
  </a>
</h1>

<nav>
  <ul>
    <li>
      <a href="//blog.byronjsmith.com/about.html"
         >
        About
      </a>
    </li>
    <li>
      <a href="//blog.byronjsmith.com/category/biology.html"
        class="active"
>
        Biology
      </a>
    </li>
    <li>
      <a href="//blog.byronjsmith.com/category/computing.html"
>
        Computing
      </a>
    </li>
    <li>
      <a href="//blog.byronjsmith.com/category/data.html"
>
        Data
      </a>
    </li>
    <li>
      <a href="//blog.byronjsmith.com/category/education.html"
>
        Education
      </a>
    </li>
    <li>
      <a href="//blog.byronjsmith.com/category/misc.html"
>
        Misc.
      </a>
    </li>
  </ul>
</nav>
<!-- /page-chrome -->
    </header>
    <div id="page-content">
<article itemscope itemtype="https://schema.org/BlogPosting">
  <header>
    <h1 id="page-title" class="headline" itemprop="headline">
      <a href="//blog.byronjsmith.com/microbial-species.html" rel="bookmark"
         title="Permalink to Do bacterial species exist?">
        Do bacterial species exist?
      </a>
    </h1>
  </header>
<footer class="article-info">

  <div class="published">
    <span class="info-tag">published:</span>
    <span class="info-value" itemprop="datePublished">
      <abbr title="2013-08-06 18:30:00-04:00">
        Tuesday, August 6, 2013
      </abbr>
    </span>
  </div>



  <div class="category">
    <span class="info-tag">category:</span>
    <span class="info-value" itemprop="articleSection">
      <a href="//blog.byronjsmith.com/category/biology.html">
        Biology
      </a>
    </span>
  </div>

  <div class="tags">
    <span class="info-tag">tags:</span>
    <span class="info-value" itemprop="keywords">
      <ul class="article-tag-list">
        <li>
          <a href="//blog.byronjsmith.com/tag/microbiology.html">
            microbiology
          </a>
        </li>
        <li>
          <a href="//blog.byronjsmith.com/tag/species.html">
            species
          </a>
        </li>
        <li>
          <a href="//blog.byronjsmith.com/tag/semantics.html">
            semantics
          </a>
        </li>
        <li>
          <a href="//blog.byronjsmith.com/tag/rant.html">
            rant
          </a>
        </li>
      </ul>
    </span>
  </div><!-- /.tags -->

</footer>
  <div class="article-content" itemprop="articleBody">
    <p>Any of my friends or colleagues who have had the "pleasure" of talking about
science with me for more than a few hours know that I am prepared, at the drop
of a hat, to rant extensively about several standing debates in biology which I
consider merely semantic.  For instance:</p>
<blockquote>
<p><strong>Q</strong>: Are viruses alive?</p>
<p><strong>A</strong>: <em>Who CARES!?</em>  Viruses do what they do.  Cellular organisms do
something else. What difference does it make if we decide to allow our
middle-schoolers to draw little dotted lines around animals, bacteria, <em>and</em>
viruses?  And I don't even want to <em>hear</em> the word "prion".</p>
</blockquote>
<p>I have a similar level of disdain for people who try to decide on a single
definition for "species".   Ultimately I am a pluralist: the definition should
be tailored to the scientific question.  Paleontologists, you have your
morphological species concept, because what <em>else</em> do stony
fossils allow?  Are you studying speciation, you can keep the biological species
concept, just be prepared for the inevitable craziness when fairly divergent
plants hybridize or genes are transferred horizontally.  Macro-evolution folks,
don't give up on the phylogenetic species concept.</p>
<p>As an aside, I have no tolerance for arbitrary definitions which do not admit
to their arbitrariness.  Since I have yet to be convinced otherwise, this has
precluded my acceptance of higher taxonomic divisions.  If someone can give me
a non-arbitrary definition for genus, order, family, phylum, etc. I would love
to know what it is.  As far as I am concerned they're just a naming system for
monophyletic groups which pander to dead systematists.</p>
<h2>Most species concepts are not applicable or arbitrary</h2>
<p>For us microbiologists, several
<a href="http://dx.doi.org/10.1146/annurev.micro.56.012302.160634">species concepts</a>
are in use.  The genetic species concept groups organisms whose full genomes or
selected genes have diverged minimally.  This definition is applied in
operational taxonomic unit (OTU) based analysis, and indirectly in whole genome
DNA-DNA hybridization, the traditional determination of bacterial species.</p>
<p>OTU thresholds are certainly arbitrary; The definition fully acknowledges that
fact.  But the OTU concept is particularly useful for culture independent
methods, since genetic distance is perhaps the only measure of familial
resemblance available.  Just as the biological species concept doesn't do much
good for a paleontologist who lacks live, mating specimens, a morphological
species concept is helpless when faced with sequence divorced from physiology.</p>
<p>When in possession of pure cultures, however, and therefore the luxury of
phenotypic data, a physiological (which I'm using synonymously with
"morphological") species concept is viable.  Unfortunately, application of the
morphogical definition depends critically on the exhaustivity of our search for
physiological differences and an arbitrary delineation between groups.</p>
<p>The phylogenetic species concept is equally impotent.  I challenge the reader
to devise a monophyletic "basic unit" for any
<a href="http://en.wikipedia.org/wiki/Asexual_reproduction">agamogenic</a> organism which
does not classify every daughter as a new species.</p>
<p>Despite the lack of canonical sex in microbes, an unsurprising (if not
justifiable) urge to model ourselves after animal ecologists has driven some
microbiologists to consider a biological species concept as well.  With many
microbes partaking in some form of recombination through transduction,
transformation, or conjugation, it seems feasible to group those individuals
which are capable of exchanging genetic material into a single species.
Unfortunately this definition leaves behind all those clades which are <em>not</em>
known to horizontally transfer genes.  What's more, it fails the "arbitrariness
test": how much recombination is required before two individuals share a
species?</p>
<h2>The ecological species concept saves the day</h2>
<p>So if everything else is ultimately arbitrary, what's left?  Originally
formulated by Leigh Van Valen
(<a href="http://www.jstor.org/stable/1219444">1976</a>),
the ecological species concept has been distilled down to
(<a href="http://dx.doi.org/10.1111/j.1420-9101.2009.01897.x">Martinez-Gordillo, 2010</a>):</p>
<blockquote>
<p>A species is a set of [related] organisms exploiting
(or adapted to) a single niche.</p>
</blockquote>
<p>Now that's interesting!  The ecological concept flips the
<a href="http://en.wikipedia.org/wiki/Competitive_exclusion_principle">competitive exclusion principle</a>
upside down.  This definition sets the threshold for familial resemblance at
the point where two lineages are no longer in the same niche.  While there are
still some rough edges, namely questions about what constitutes a niche, this
definition appears to pass the test of arbitrariness and makes a satisfying
connection between taxonomy and ecology.</p>
<p>Despite their inherent differences in criteria, and in my opinion
defensibility, all of these species concepts (genetic, morphological,
biological, and ecological) are getting at the same core idea: what makes one
organism more "like" another?  What keeps the diversity of life clumped
together into identifiable units?  There is a certain harmony to these species
concepts.  Whether <strong>recombination</strong> or <strong>ecological</strong> processes, we expect the
end result to be <strong>genetic</strong> similarity which will manifest itself as
<strong>physiological</strong> clustering.</p>
<h2>Are species empirically real?</h2>
<p>But wait.  This is certainly begging the question.  Does the microbial world
<em>actually</em> have this assumed genetic and physiological clustering?  Finally, a
hypothesis to be tested!</p>
<p>Hanage <em>et al.</em> have (<a href="http://dx.doi.org/10.1098%2Frstb.2006.1917">2006</a>)
summarized the current literature on genomic clustering and, in a second paper
(<a href="http://dx.doi.org/10.1126/science.1159388">2009</a>), the models which may
explain the observations.  The data is somewhat ambiguous.  <strong>Figure 1</strong> from
the 2009 paper demonstrates several of the problems facing an empirically
described microbial species.</p>
<p><a href="http://www.sciencemag.org/content/323/5915/741/F1.expansion.html"><strong>Figure 1:</strong></a></p>
<p><img alt="Two examples exhibiting conflicts with species concepts in microbes." src="http://www.sciencemag.org/content/323/5915/741/F1.large.jpg"></p>
<p>First, and most importantly, <strong>1A</strong> shows that genetic distance between and
within clusters of <em>Streptococcus</em> isolates is not consistent.  Current species
designations are indicated.  Clearly, the level of genetic distance within any
visually (indicated by terminal node color) or <em>a priori</em> designated species
varies by a factor of 5 or more.  What's more, <strong>1B</strong> demonstrates that while
ecological preference (terminal node color) is fairly correlated with phylogeny
in <em>Vibrio</em>, some clades lack this association, specifically <em>V. splendidus</em>.</p>
<p>The fractal-like clustering in the <em>Streptococcus</em> phylogeny makes me wonder if
we're not seeing the result of numerous intense bottlenecks or selective sweeps
which occur stochastically.  And, while the ecological-phylogenetic mapping is
fairly consistent, a point in favor of the species concept, I am curious if
stabilizing selection and/or recombination models explain this clustering
better than other, less interesting processes (e.g. gradual niche migration).
I'm also concerned about problems with sampling.  If there were genotypes which
bridged the clusters would we know it?  And, like every database dependent
analysis in microbiology, a bias for pathogens could be misleading us here as
well.</p>
<h2><a href="http://en.wikipedia.org/wiki/Betteridge's_law_of_headlines">So...</a></h2>
<p>Despite a exhaustive debate over the semantics of asexual species, surprisingly
little data has has been collected to show that bacteria even exhibit more than
trivial clustering.  Several mechanisms have been proposed for maintaining
genetic conformity, namely recombination and niche specificity.  Some of this
research has aimed to find parallels with plant and animal ecology, but I would
suggest we lose little by discarding the species concept entirely.</p>
<p>I can understand the need to define morpho-species when diagnosing disease, or
the utility of OTU based approaches in molecular surveys.  I'm also not
suggesting that we should stop categorizing strains into species; the
convenience for communicating microbiology is undeniable.  But by remembering
that most species divisions are arbitrary, and that every microbe is a new
lineage, we're one step closer to escaping the chains of the old microbiology.</p>
  </div>
  
</article>
<div class="comments">
  <h2>Comments</h2>
  <div id="disqus_thread"></div>
</div>
<script type="text/javascript">
    var disqus_shortname = 'byronjsmithblog';
    var disqus_identifier = 'microbial-species.html';
    var disqus_title = 'Do bacterial species exist?';
    var disqus_url = '//blog.byronjsmith.com/microbial-species.html';
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the
  <a href="https://disqus.com/?ref_noscript" rel="nofollow">
    comments powered by Disqus.
  </a>
</noscript>



    </div>
    <footer id="bottom">
<div id="extras">
  <div class="blogroll">
    <h2>blogroll</h2>
    <ul>
      <li><a href="http://ivory.idyll.org/blog/">Living in an Ivory Basement</a></li>
      <li><a href="https://www.johndcook.com/blog/">The Endeavor</a></li>
      <li><a href="https://statmodeling.stat.columbia.edu/">Andrew Gelman (and others)</a></li>
    </ul>
  </div>
  <div class="social">
    <h2>social</h2>
    <ul>
      <li><a  href="//blog.byronjsmith.com/feeds/all.atom.xml"
          type="application/atom+xml"
          rel="alternate">
        atom feed
      </a></li>
      <li><a href="https://twitter.com/ByronJSmith">twitter</a></li>
      <li><a href="https://linkedin.com/profile/view?id=76273001">linkedin</a></li>
      <li><a href="https://github.com/bsmith89">github</a></li>
    </ul>
  </div>
</div><!-- /#extras -->

<div id="site-info">
  <p id="site-copyright">
    The contents of this blog are licensed <a rel="license" href="https://creativecommons.org/licenses/by/4.0"><img title="CC-BY" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>
  </p>
  <p id="theme-info">
    This <a href="https://github.com/bsmith89/blog-theme">theme</a>
    was originally created for
    <a href="http://blog.byronjsmith.com">blog.byronjsmith.com</a>
    and is powered by <a href="https://getpelican.com">Pelican</a>
  </p>
</div>
    </footer>
  </div>

<!-- #mathjax-scripts -->
<!-- Using MathJax, with the delimiters $ -->
<!-- Conflict with pygments for the .mo and .mi -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
  "HTML-CSS": {
  styles: {
  ".MathJax .mo, .MathJax .mi": {color: "black ! important"}}
  },
  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']],processEscapes: true}
  });
  MathJax.Hub.Register.StartupHook("HTML-CSS Jax Ready",function () {
  var VARIANT = MathJax.OutputJax["HTML-CSS"].FONTDATA.VARIANT;
  VARIANT["normal"].fonts.unshift("MathJax_SansSerif");
  VARIANT["bold"].fonts.unshift("MathJax_SansSerif-bold");
  VARIANT["italic"].fonts.unshift("MathJax_SansSerif-italic");
  VARIANT["-tex-mathit"].fonts.unshift("MathJax_SansSerif-italic");
  });
  MathJax.Hub.Register.StartupHook("SVG Jax Ready",function () {
  var VARIANT = MathJax.OutputJax.SVG.FONTDATA.VARIANT;
  VARIANT["normal"].fonts.unshift("MathJax_SansSerif");
  VARIANT["bold"].fonts.unshift("MathJax_SansSerif-bold");
  VARIANT["italic"].fonts.unshift("MathJax_SansSerif-italic");
  VARIANT["-tex-mathit"].fonts.unshift("MathJax_SansSerif-italic");
  });
</script>
<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML"></script>
<!-- /#mathjax-scripts -->


</body>
</html>